lib/localenv/clusterenv.go

```diff
@@ -71,6 +64,12 @@ func (r *LocalEnvironment) NewClusterEnvironment(opts ...ClusterEnvironmentOptio
 	if err != nil && !trace.IsNotFound(err) {
 		return nil, trace.Wrap(err)
 	}
+	ctx, cancel := context.WithTimeout(context.TODO(), defaults.AuditLogClientTimeout)
+	defer cancel()
+	auditLog, err := r.AuditLog(ctx)
+	if err != nil && !trace.IsNotFound(err) && !trace.IsAccessDenied(err) {
```

Comment:
Why is access denied also being filtered out here?

Reply:
This does not work on regular nodes:
```
2020-08-04T17:30:00Z WARN [LOCAL]     Failed to create audit log. error:[
ERROR REPORT:
Original Error: *trace.AccessDeniedError access denied to perform action "create" on "user"
Stack Trace:
	/go/_/src/github.com/gravitational/gravity/vendor/github.com/gravitational/teleport/lib/services/role.go:1865 github.com/gravitational/gravity/vendor/github.com/gravitational/teleport/lib/services.RoleSet.CheckAccessToRule
	...
```